src/com/twitter/intellij/pants/execution/PantsMakeBeforeRun.java

```diff
@@ -250,23 +259,22 @@ public boolean executeTask(
       }
       catch (Exception e) {
         showPantsMakeTaskMessage(e.getMessage(), ConsoleViewContentType.ERROR_OUTPUT, currentProject);
-        return Pair.create(false, Optional.empty());
+        return PantsExecuteTaskResult.emptyFailure();
       }
     }
 
     /* Goals and targets section. */
-    commandLine.addParameters("export-classpath", "compile");
-    for (String targetAddress : targetAddressesToCompile) {
-      commandLine.addParameter(targetAddress);
-    }
+    commandLine.addParameters(Lists.newArrayList(tasks));
+    commandLine.addParameters(Lists.newArrayList(targetAddresses));
```

Comment:
why not just `commandLine.addParameters(tasks)`?

Reply:
Thanks for catching that. Changed.